charts/llmisvc-resources/templates/serviceaccount.yaml

```
@@ -0,0 +1,17 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "llm-isvc-resources.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "llm-isvc-resources.labels" . | nindent 4 }}
+    app.kubernetes.io/component: controller
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)